integration_tests/suite/base/test_parking_lot_extension.py

```diff
@@ -114,6 +114,49 @@ def test_associate_when_exten_is_pattern(parking_lot, extension):
     response.assert_status(400)
 
 
+@fixtures.extension(exten='600', context=CONTEXT)
+@fixtures.extension(exten='400', context=CONTEXT)
+@fixtures.parking_lot(slots_start='701', slots_end='750')
+@fixtures.parking_lot(slots_start='801', slots_end='850', wazo_tenant=SUB_TENANT)
+def test_associate_when_overlapping_slots(
+    exten_new_parking, exten_existing_parking, existing_parking, _
+):
+    def assert_association_fails(start, end):
+        with fixtures.parking_lot(slots_start=start, slots_end=end) as parking_lot:
+            response = (
+                confd.parkinglots(parking_lot['id'])
+                .extensions(exten_new_parking['id'])
+                .put()
+            )
+            response.assert_match(400, e.extension_conflict())
+
+    with a.parking_lot_extension(existing_parking, exten_existing_parking):
+        # same range
+        assert_association_fails('701', '750')
+        # overlap end
+        assert_association_fails('750', '850')
+        # overlap end -1
+        assert_association_fails('749', '850')
+        # overlap start
+        assert_association_fails('601', '701')
+        # overlap start +1
+        assert_association_fails('601', '702')
+        # overlap inner
+        assert_association_fails('710', '711')
+        # overlap outer
+        assert_association_fails('601', '850')
+
+        # check there is no conflict with parking_lot from other tenant
+        parameters = {
+            'slots_start': '801',
+            'slots_end': '850',
+        }
+
+        response = confd.parkinglots.post(**parameters)
+        response.assert_created('parkinglots')
```

Comment:
The test is wrong, the validator won't be executed for creation
It will be executed only when:
- associate parking lot to extension
- edit parking lot

I'll fix it later